docs/index.rst

```diff
@@ -61,3 +61,30 @@ Features
     how-to
     how-it-works
     changelog
+
+Known limitations
+-----------------
+pytest-xdist have some limitation that may be supported in pytest but can't be supported in pytest-xdist
+
+Order and amount of test must be constant
```

Comment:
```suggestion
Order and amount of test must be consistent
```

I think "consistent" makes more sense here, but I'm not a native speaker so I'm not 100% sure.

Reply:
you are right... Will add it in my next commit